extensions/src/main/java/org/pentaho/platform/web/servlet/UploadFileUtils.java

```diff
@@ -16,7 +16,7 @@
 import com.ice.tar.TarEntry;
 import com.ice.tar.TarInputStream;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.fileupload2.core.FileItem;
```

Comment:
There are two "Zip Slip" CodeQL warnings on this file; as there is code to validate that possible problem (we are invoking method 'validateZipSlip()'), one could add the proper Sonar Ignore annotation to avoid these false positives.

Reply:
As of now CodeQL has configurations to exclude the file from scanning, but I didn't come across a solution to suppress inline warnings